accname/index.html

```diff
@@ -554,9 +554,8 @@ <h4>Computation steps</h4>
                   </div>
                 </li>
                 <li id="comp_embedded_control">
-                  <span id="step2C"><!-- Don't link to this legacy numbered ID. --></span><em>Embedded Control:</em> Otherwise, if the <code>current node</code> is a control embedded within the label
-                  (e.g. any element directly referenced by <code>aria-labelledby</code>) for another <a class="termref">widget</a>, where the user can adjust the embedded control's value, then return
-                  the embedded control as part of the text alternative in the following manner:
+                  <span id="step2C"><!-- Don't link to this legacy numbered ID. --></span><em>Embedded Control:</em> Otherwise, if the <code>current node</code> is a user-adjustable control embedded
+                  within the label, and the control is not the element being named or described, then return the embedded control's value as part of the text alternative in the following manner:
```

Comment:
Nit: take or leave. "as part of the text alternative" makes it sound (a little bit) like you're overriding how this returned value would be included in the over text alternative computation... I think  it stands without that clause, but feel free to keep it and resolve this comment if you disagree.

```suggestion
                  within the label, and the control is not the element being named or described, then return the embedded control's value in the following manner:
```

Reply:
it might but I think it was part of the feedback to add it in the first place.